### PR TITLE
#155972746 allow email sessions

### PR DESCRIPTION
**Pull request by Ashaba**
#### What does this PR do?
It ensures sessions are on email address
#### Description of Task to be completed?
The session key of name was being used in some instances which caused bugs and instead a uniform email is going to be used since they are unique.
#### How should this be manually tested?
pytest or save a team

### CODE REVIEW

**Comment by muhallan** [comment]
Typo in the PR description

**Comment by muhallan** [comment]
LGTM

File: application/views.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from flask import Blueprint, render_template, request
+from flask import Blueprint, render_template, request, session
```
Comment (eric-elem) [inline application/views.py]: `session` is imported but **not** used.

Comment (eric-elem) [reply application/views.py]: The file `test/test_db.sqlite` shouldn't be pushed.

File: .gitignore [hunk 2]
```diff
@@ -14,6 +14,8 @@ __pycache__/
 
 .pytest_cache
 
+test/test_db.sqlite/
```
Comment (eric-elem) [inline .gitignore]: This isn't working because of the `/` at the end. I see `test/test_db.sqlite` has still been pushed.

Comment (muhallan) [reply .gitignore]: He should actually delete it because it's already taken care of by `*.sqlite`

Comment (muhallan) [reply .gitignore]: @Ashaba I am going to do for you this, so that I merge this PR

**Comment by eric-elem** [comment]
LGTM

**Pull request merged**